activiti-api-task-runtime/src/main/java/org/activiti/api/task/runtime/TaskRuntime.java

```diff
@@ -115,4 +117,13 @@
     List<VariableInstance> variables(GetTaskVariablesPayload getTaskVariablesPayload);
 
     void setVariables(SetTaskVariablesPayload setTaskVariablesPayload);
+    
+    void addCandidateUsers(CandidateUsersPayload candidateUsersPayload);
```

Comment:
@CTI777 here.. you can only call this methods if the logged in user is assigned to the task, if not it should throw an exception

Reply:
Actually it is like this, and checks will be applied in the Implementation method. candidateUsersPayload will contain list of users which should be added as candidates...
What exactly should i do here? Add a comment?
@salaboy

Reply:
@CTI777 no action here.. it was only a note for the implementation